### ISSUE TITLE
[BUG]:  Set Google Chrome as External Browser For RSSGuard Flatpak

### ISSUE THREAD

**Issue by richardstevenhack** [martinrotter/rssguard#1539]
### Brief description of the issue

I have been using Firefox as the external browser in RSSGuard for quite a while. However, Firefox has had a tab crashing problem on my system lately, so I switched my default browser to Google Chrome.

I now need to set Chrome as the external browser in my Flatpak installed RSSGuard. I have tried a variety of ways based on the way the Firefox was set up and I can not get it to work.

**Basically what I need is for someone using the Flatpak version of RSSGuard to walk me through the proper settings for Google Chome as the external browser.** 

### How to reproduce the bug?

1) Open Settings, Network Web & Tools, External Web Browser.
2) Set Web Browser Executable to: "/usr/bin flatpak-spawn" 
3) Set Parameters to: --host firefox "%1"
4) The above works with Firefox.
5) Leave the Web Browser Executable as it is above.
6) Set Parameters to: --host /usr/bin/google-chrome-stable "%1"
7) Running: flatpak-spawn --host "/usr/bin/google-chrome-stable" from the command line execute the browser as expected.
8) Open RSSGuard, right click on a news item, and select "Open In External Browser"
9) RSSGuard reports "RSS Guard was unable to launch your web browser with the given URL, you need to open the below website URL in your web browser manually"

### What was the expected result?

Expected result is the Google Chrome browser opens with the selected news item.

### What actually happened?

 RSSGuard reports "RSS Guard was unable to launch your web browser with the given URL, you need to open the below website URL in your web browser manually"

### Debug log

time="     0.105" type="debug" -> core: Initializing settings in '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/config/config.ini' (non-portable way).
time="     0.105" type="debug" -> core: Starting to load active localization. Desired localization is 'en_US'.
time="     0.105" type="debug" -> core: Application localization 'en_US' loaded successfully, specifically sublocalization 'en_US' was loaded.
time="     0.105" type="warning" -> core: Qt localization 'en_US' WAS NOT loaded successfully.
time="     0.302" type="debug" -> database: File-based SQLite database connection 'DatabaseFactory' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.302" type="debug" -> database: File-based SQLite database has version '8'.
time="     0.302" type="debug" -> gui: Available icon theme paths: QList(/app/share/icons, /usr/share/icons, /usr/share/runtime/share/icons, /run/host/user-share/icons, /run/host/share/icons, :/icons, :/graphics, /home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/icons, /app/bin/icons)
time="     0.302" type="debug" -> gui: Installed icon themes are: '', 'Breeze_Light', 'breeze', 'breeze-dark', 'breeze_cursors', 'hicolor', 'Adwaita', 'Breeze', 'Breeze Dark', 'Faenza', 'Numix'
time="     0.302" type="debug" -> gui: Loading icon theme 'Breeze'.
time="     0.302" type="debug" -> gui: Found path of base skin: ':/skins/nudus-base'.
time="     0.302" type="debug" -> gui: Base file ':/skins/nudus-base/html_wrapper.html' exists, using it for the skin.
time="     0.302" type="debug" -> gui: Local file ':/skins/nudus-light/html_style.css' exists, using it for the skin.
time="     0.302" type="debug" -> gui: Base file ':/skins/nudus-base/html_enclosure_image.html' exists, using it for the skin.
time="     0.302" type="debug" -> gui: Base file ':/skins/nudus-base/html_single_message.html' exists, using it for the skin.
time="     0.302" type="debug" -> gui: Base file ':/skins/nudus-base/html_enclosure_every.html' exists, using it for the skin.
time="     0.302" type="debug" -> gui: Local file ':/skins/nudus-light/qt_style.qss' exists, using it for the skin.
time="     0.302" type="debug" -> gui: Base file ':/skins/nudus-base/html_adblocked.html' exists, using it for the skin.
time="     0.303" type="debug" -> gui: OS defines color scheme: 'Qt::ColorScheme::Light'.
time="     0.303" type="debug" -> gui: Setting style: ''.
time="     0.303" type="debug" -> gui: Skin 'nudus-light' loaded. Lite: 'false'.
time="     0.303" type="warning" -> core: User CSS style was not provided in file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/web/user-styles.css'.
time="     0.303" type="debug" -> network: Persistent web data storage path: '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/web/storage'.
time="     0.303" type="debug" -> network: Disabling application-wide proxy completely.
time="     0.303" type="debug" -> core: Platform: 'xcb'.
time="     0.303" type="debug" -> core: SQLite version: '3.46.1'.
time="     0.306" type="debug" -> core: OpenSSL version: 'OpenSSL 3.3.2 3 Sep 2024'.
time="     0.306" type="debug" -> core: OpenSSL supported: 'true'.
time="     0.306" type="debug" -> core: Global thread pool has 32 threads.
time="     0.306" type="debug" -> core: Setting locale for LC_NUMERIC to C as libmpv requires it.
time="     0.306" type="debug" -> core: Instantiated class  'Application'.
time="     0.307" type="debug" -> database: SQLite database connection 'MessagesModel' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.308" type="debug" -> message-model: Repopulated model, SQL statement is now:
 'SELECT Messages.id, Messages.is_read, Messages.is_important, Messages.is_deleted, Messages.is_pdeleted, Messages.feed, Messages.title, Messages.url, Messages.author, Messages.date_created, Messages.contents, Messages.enclosures, Messages.score, Messages.account_id, Messages.custom_id, Messages.custom_hash, Feeds.title, Feeds.is_rtl, CASE WHEN LENGTH(Messages.enclosures) > 10 THEN 'true' ELSE 'false' END AS has_enclosures, (SELECT GROUP_CONCAT(Labels.name) FROM Labels WHERE Messages.labels LIKE '%.' || Labels.custom_id || '.%') as msg_labels, Messages.labels FROM Messages LEFT JOIN Feeds ON Messages.feed = Feeds.custom_id AND Messages.account_id = Feeds.account_id WHERE 0 > 1;'.
time="     0.344" type="debug" -> core: Auto-download timer started with interval 10000 ms.
time="     0.344" type="debug" -> core: Creating FeedDownloader singleton.
time="     0.354" type="debug" -> gui: Creating main application form in thread: '2'.
time="     0.358" type="debug" -> gui: Current row changed - proxy 'QModelIndex(-1,-1,0x0,QObject(0x0))', source 'QModelIndex(-1,-1,0x0,QObject(0x0))'.
time="     0.358" type="debug" -> gui: Article list header geometries changed.
time="     0.361" type="debug" -> network: Settings of BaseNetworkAccessManager loaded.
time="     0.376" type="debug" -> gui: Article list header geometries changed.
time="     0.378" type="debug" -> gui: Article list header geometries changed.
time="     0.380" type="debug" -> gui: Article list header geometries changed.
time="     0.382" type="debug" -> gui: Article list header geometries changed.
time="     0.396" type="debug" -> gui: Creating tray icon menu.
time="     0.413" type="debug" -> CTRL is NOT pressed while sorting articles - sorting with standard mode.
time="     0.413" type="debug" -> CTRL is NOT pressed while sorting articles - sorting with standard mode.
time="     0.413" type="debug" -> message-model: Repopulated model, SQL statement is now:
 'SELECT Messages.id, Messages.is_read, Messages.is_important, Messages.is_deleted, Messages.is_pdeleted, Messages.feed, Messages.title, Messages.url, Messages.author, Messages.date_created, Messages.contents, Messages.enclosures, Messages.score, Messages.account_id, Messages.custom_id, Messages.custom_hash, Feeds.title, Feeds.is_rtl, CASE WHEN LENGTH(Messages.enclosures) > 10 THEN 'true' ELSE 'false' END AS has_enclosures, (SELECT GROUP_CONCAT(Labels.name) FROM Labels WHERE Messages.labels LIKE '%.' || Labels.custom_id || '.%') as msg_labels, Messages.labels FROM Messages LEFT JOIN Feeds ON Messages.feed = Feeds.custom_id AND Messages.account_id = Feeds.account_id WHERE 0 > 1 ORDER BY Messages.date_created ASC, Messages.is_read ASC;'.
time="     0.415" type="debug" -> core: Showing the main window when the application is starting.
time="     0.422" type="debug" -> gui: Article list header geometries changed.
time="     0.422" type="debug" -> gui: Article list header geometries changed.
time="     0.423" type="debug" -> gui: Article list header geometries changed.
time="     0.424" type="debug" -> database: SQLite database connection 'FeedReader' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.424" type="debug" -> core: Checking for plugins in '/app/bin/../lib/rssguard'.
time="     0.426" type="debug" -> core: Plugin '/app/lib/rssguard/librssguard-feedly.so' loaded.
time="     0.428" type="debug" -> core: Plugin '/app/lib/rssguard/librssguard-gmail.so' loaded.
time="     0.430" type="debug" -> core: Plugin '/app/lib/rssguard/librssguard-greader.so' loaded.
time="     0.431" type="debug" -> core: Plugin '/app/lib/rssguard/librssguard-nextcloud.so' loaded.
time="     0.432" type="debug" -> core: Plugin '/app/lib/rssguard/librssguard-reddit.so' loaded.
time="     0.435" type="debug" -> core: Plugin '/app/lib/rssguard/librssguard-standard.so' loaded.
time="     0.436" type="debug" -> core: Plugin '/app/lib/rssguard/librssguard-ttrss.so' loaded.
time="     0.436" type="debug" -> database: SQLite database connection 'FeedlyEntryPoint' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.437" type="debug" -> database: SQLite database connection 'GmailEntryPoint' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.437" type="debug" -> database: SQLite database connection 'GreaderEntryPoint' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.437" type="debug" -> database: SQLite database connection 'NextcloudServiceEntryPoint' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.437" type="debug" -> database: SQLite database connection 'RedditEntryPoint' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.437" type="debug" -> database: SQLite database connection 'StandardServiceEntryPoint' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.437" type="debug" -> core: Destructing 'std-rss' plugin.
time="     0.438" type="debug" -> database: SQLite database connection 'StandardServiceRoot' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.438" type="debug" -> core: Custom ID of feed when loading from DB is '10'.
time="     0.438" type="debug" -> core: Custom ID of feed when loading from DB is '11'.
time="     0.438" type="debug" -> core: Custom ID of feed when loading from DB is '12'.
time="     0.438" type="debug" -> core: Custom ID of feed when loading from DB is '13'.
time="     0.438" type="debug" -> core: Custom ID of feed when loading from DB is 'new'.
time="     0.438" type="debug" -> core: Custom ID of feed when loading from DB is '26'.
time="     0.438" type="debug" -> core: Custom ID of feed when loading from DB is '27'.
time="     0.438" type="debug" -> core: Custom ID of feed when loading from DB is '28'.
time="     0.438" type="debug" -> core: Custom ID of feed when loading from DB is '29'.
time="     0.438" type="debug" -> core: Custom ID of feed when loading from DB is '30'.
time="     0.438" type="debug" -> core: Custom ID of feed when loading from DB is '31'.
time="     0.438" type="debug" -> core: Custom ID of feed when loading from DB is '32'.
time="     0.438" type="debug" -> core: Custom ID of feed when loading from DB is '33'.
time="     0.439" type="debug" -> core: Custom ID of feed when loading from DB is '34'.
time="     0.439" type="debug" -> core: Custom ID of feed when loading from DB is '35'.
time="     0.439" type="debug" -> core: Custom ID of feed when loading from DB is '36'.
time="     0.439" type="debug" -> core: Custom ID of feed when loading from DB is '37'.
time="     0.439" type="debug" -> core: Custom ID of feed when loading from DB is '38'.
time="     0.439" type="debug" -> core: Custom ID of feed when loading from DB is '39'.
time="     0.439" type="debug" -> core: Custom ID of feed when loading from DB is '40'.
time="     0.439" type="debug" -> core: Custom ID of feed when loading from DB is '41'.
time="     0.439" type="debug" -> core: Custom ID of feed when loading from DB is '42'.
time="     0.439" type="debug" -> core: Custom ID of feed when loading from DB is '43'.
time="     0.439" type="debug" -> core: Custom ID of feed when loading from DB is '44'.
time="     0.439" type="debug" -> core: Custom ID of feed when loading from DB is '45'.
time="     0.439" type="debug" -> core: Custom ID of feed when loading from DB is '46'.
time="     0.439" type="debug" -> core: Custom ID of feed when loading from DB is '47'.
time="     0.439" type="debug" -> core: Custom ID of feed when loading from DB is '58'.
time="     0.440" type="debug" -> core: Custom ID of feed when loading from DB is '59'.
time="     0.440" type="debug" -> core: Custom ID of feed when loading from DB is '60'.
time="     0.440" type="debug" -> core: Custom ID of feed when loading from DB is '61'.
time="     0.440" type="debug" -> core: Custom ID of feed when loading from DB is '62'.
time="     0.440" type="debug" -> core: Custom ID of feed when loading from DB is '63'.
time="     0.440" type="debug" -> core: Custom ID of feed when loading from DB is '64'.
time="     0.440" type="debug" -> core: Custom ID of feed when loading from DB is '86'.
time="     0.440" type="debug" -> core: Custom ID of feed when loading from DB is '87'.
time="     0.440" type="debug" -> core: Custom ID of feed when loading from DB is '88'.
time="     0.440" type="debug" -> core: Custom ID of feed when loading from DB is '89'.
time="     0.440" type="debug" -> core: Custom ID of feed when loading from DB is '90'.
time="     0.440" type="debug" -> core: Custom ID of feed when loading from DB is '91'.
time="     0.440" type="debug" -> core: Custom ID of feed when loading from DB is '92'.
time="     0.441" type="debug" -> core: Custom ID of feed when loading from DB is '93'.
time="     0.441" type="debug" -> core: Custom ID of feed when loading from DB is '94'.
time="     0.441" type="debug" -> core: Custom ID of feed when loading from DB is '95'.
time="     0.441" type="debug" -> core: Custom ID of feed when loading from DB is '96'.
time="     0.441" type="debug" -> core: Custom ID of feed when loading from DB is '97'.
time="     0.441" type="debug" -> core: Custom ID of feed when loading from DB is '99'.
time="     0.441" type="debug" -> core: Custom ID of feed when loading from DB is '100'.
time="     0.441" type="debug" -> core: Custom ID of feed when loading from DB is '101'.
time="     0.441" type="debug" -> core: Custom ID of feed when loading from DB is '102'.
time="     0.441" type="debug" -> core: Custom ID of feed when loading from DB is '103'.
time="     0.441" type="debug" -> core: Custom ID of feed when loading from DB is '104'.
time="     0.441" type="debug" -> core: Custom ID of feed when loading from DB is '105'.
time="     0.442" type="debug" -> core: Custom ID of feed when loading from DB is '106'.
time="     0.442" type="debug" -> core: Custom ID of feed when loading from DB is '107'.
time="     0.442" type="debug" -> core: Custom ID of feed when loading from DB is '108'.
time="     0.442" type="debug" -> core: Custom ID of feed when loading from DB is '109'.
time="     0.442" type="debug" -> core: Custom ID of feed when loading from DB is '110'.
time="     0.442" type="debug" -> core: Custom ID of feed when loading from DB is '111'.
time="     0.442" type="debug" -> core: Custom ID of feed when loading from DB is '112'.
time="     0.442" type="warning" -> core: Feed 'new' is loose, skipping it.
time="     0.442" type="debug" -> database: SQLite database connection 'RecycleBin' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.450" type="debug" -> database: SQLite database connection 'ImportantNode' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.451" type="debug" -> database: SQLite database connection 'UnreadNode' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.452" type="debug" -> database: SQLite database connection 'LabelsNode' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.452" type="debug" -> database: SQLite connection 'StandardServiceRoot' is already active.
time="     0.452" type="debug" -> database: SQLite database connection 'StandardServiceRoot' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.453" type="debug" -> database: SQLite database connection 'TtRssServiceEntryPoint' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.453" type="debug" -> gui: User wants to have tray icon.
time="     0.454" type="warning" -> gui: Showing tray icon with little delay.
time="     0.454" type="debug" -> gui: Feed list item expanded - rhack (RSS/ATOM/JSON)
time="     0.454" type="debug" -> gui: Feed list item expanded - Original Feeds
time="     0.454" type="debug" -> gui: Feed list item expanded - Imported Feeds
time="     0.454" type="debug" -> core: No execution message received from other app instances.
time="     0.500" type="debug" -> gui: Hovered link: 'QUrl()'.
time="     3.357" type="warning" -> gui: Tray icon is available, showing now.
time="     3.358" type="debug" -> gui: Creating SystemTrayIcon instance.
time="     3.358" type="debug" -> gui: Showing tray icon immediately.
time="     3.363" type="debug" -> gui: Tray icon displayed.
time="    25.717" type="debug" -> core: Syncing settings.
time="    25.720" type="debug" -> gui: Saving size for dialog 'FormSettings'.
time="    25.720" type="debug" -> gui: Destroying FormSettings distance.
time="    28.555" type="debug" -> CTRL is NOT pressed while sorting articles - sorting with standard mode.
time="    28.555" type="debug" -> core: Displaying messages from feeds IDs: ''106', '107', '108', '110'' and URLs: 'https://www.cyberciti.com/atom/atom.xml, https://itsfoss.com/rss/, http://linuxtoday.com/biglt.rss?x=1, https://www.reddit.com/r/linux/top.rss'.
time="    28.560" type="debug" -> message-model: Repopulated model, SQL statement is now:
 'SELECT Messages.id, Messages.is_read, Messages.is_important, Messages.is_deleted, Messages.is_pdeleted, Messages.feed, Messages.title, Messages.url, Messages.author, Messages.date_created, Messages.contents, Messages.enclosures, Messages.score, Messages.account_id, Messages.custom_id, Messages.custom_hash, Feeds.title, Feeds.is_rtl, CASE WHEN LENGTH(Messages.enclosures) > 10 THEN 'true' ELSE 'false' END AS has_enclosures, (SELECT GROUP_CONCAT(Labels.name) FROM Labels WHERE Messages.labels LIKE '%.' || Labels.custom_id || '.%') as msg_labels, Messages.labels FROM Messages LEFT JOIN Feeds ON Messages.feed = Feeds.custom_id AND Messages.account_id = Feeds.account_id WHERE Feeds.custom_id IN ('106', '107', '108', '110') AND Messages.is_deleted = 0 AND Messages.is_pdeleted = 0 AND Messages.account_id = 1 ORDER BY Messages.date_created ASC, Messages.is_read ASC;'.
time="    33.479" type="debug" -> gui: Message list got focus with reason 'Qt::MouseFocusReason'.
time="    33.479" type="debug" -> gui: Current row changed - proxy 'QModelIndex(11,6,0x55ed160e0630,MessagesProxyModel(0x55ed15e0ea20, name = MessagesProxyModel))', source 'QModelIndex(11,6,0x0,MessagesModel(0x55ed15dd4c70))'.
time="    33.479" type="debug" -> gui: Marking article as read immediately.
time="    33.480" type="debug" -> database: SQLite database connection 'StandardFeed' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="    33.484" type="debug" -> database: SQLite connection 'UnreadNode' is already active.
time="    33.484" type="debug" -> database: SQLite database connection 'UnreadNode' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="    33.485" type="debug" -> feed-model: There is request to reload feed model, reloading the 2 items individually.
time="    33.491" type="debug" -> gui: HTML image resizing took 0 miliseconds.
time="    33.503" type="debug" -> gui: Hovered link: 'QUrl()'.
time="    37.221" type="debug" -> gui: Message list got focus with reason 'Qt::PopupFocusReason'.
time="    37.221" type="debug" -> network: We are trying to open URL 'https://www.linuxtoday.com/blog/rocky-linux-9-5-released-heres-whats-new/'.
time="    37.221" type="debug" -> network: Arguments for external browser: '--host /usr/bin/google-chrome-stable "https://www.linuxtoday.com/blog/rocky-linux-9-5-released-heres-whats-new/"'.
time="    37.222" type="debug" -> network: External web browser call failed.
time="    42.132" type="debug" -> database: SQLite connection 'StandardFeed' is already active.
time="    42.132" type="debug" -> database: SQLite database connection 'StandardFeed' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="    42.137" type="debug" -> database: SQLite connection 'UnreadNode' is already active.
time="    42.137" type="debug" -> database: SQLite database connection 'UnreadNode' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="    42.138" type="debug" -> feed-model: There is request to reload feed model, reloading the 2 items individually.
time="    42.140" type="debug" -> gui: Message list got focus with reason 'Qt::ActiveWindowFocusReason'.
time="    45.857" type="debug" -> core: Saved data with auto-saver for 'CookieJar' and method 'saveCookies'.
time="    48.571" type="debug" -> gui: Main window hide event
time="    48.571" type="debug" -> gui: Article list header geometries changed.
time="    48.572" type="debug" -> gui: Main window close event
time="    55.261" type="debug" -> core: Cleaning up resources and saving application state.
time="    55.261" type="debug" -> core: Close lock was obtained safely.
time="    55.261" type="debug" -> core: Stopping StandardServiceRoot instance.
time="    55.261" type="debug" -> feed-downloader: Destroying FeedDownloader instance.
time="    55.263" type="debug" -> gui: Destroying FormMain instance.
time="    55.263" type="debug" -> gui: Destroying TabWidget instance.
time="    55.263" type="debug" -> gui: Destroying FeedMessageViewer instance.
time="    55.263" type="debug" -> gui: Destroying BaseToolBar instance.
time="    55.264" type="debug" -> network: Destroying Downloader instance.
time="    55.264" type="debug" -> network: Destroying SilentNetworkAccessManager instance.
time="    55.266" type="debug" -> gui: Destroying MessagesView instance.
time="    55.266" type="debug" -> gui: Destroying BaseToolBar instance.
time="    55.267" type="debug" -> gui: Destroying FeedsView instance.
time="    55.267" type="debug" -> gui: Destroying TabBar instance.
time="    55.267" type="debug" -> gui: Destroying StatusBar instance.
time="    55.267" type="debug" -> gui: Destroying SystemTrayIcon instance.
time="    55.282" type="debug" -> core: Destroying Application instance.
time="    55.282" type="debug" -> core: Destroying Mutex instance.
time="    55.289" type="debug" -> gui: Destroying IconFactory instance.
time="    55.289" type="debug" -> core: Destroying FeedReader instance.
time="    55.289" type="debug" -> core: Service 'feedly' will be deleted by runtime.
time="    55.289" type="debug" -> core: Service 'gmail' will be deleted by runtime.
time="    55.289" type="debug" -> core: Service 'greader' will be deleted by runtime.
time="    55.289" type="debug" -> core: Service 'nextcloud' will be deleted by runtime.
time="    55.289" type="debug" -> core: Service 'reddit' will be deleted by runtime.
time="    55.289" type="debug" -> core: Service 'std-rss' will be deleted by runtime.
time="    55.289" type="debug" -> core: Service 'tt-rss' will be deleted by runtime.
time="    55.289" type="debug" -> feed-model: Destroying FeedsModel instance.
time="    55.289" type="debug" -> feed-model: Destroying FeedsProxyModel instance
time="    55.289" type="debug" -> message-model: Destroying MessagesModel instance.
time="    55.294" type="debug" -> message-model: Destroying MessagesProxyModel instance.
core: Destructing 'feedly' plugin.
gmail: Destructing 'gmail' plugin.
core: Destructing 'greader' plugin.
gmail: Destructing 'nextcloud' plugin.
gmail: Destructing 'nextcloud' plugin.
core: Destructing 'std-rss' plugin.
gmail: Destructing 'tt-rss' plugin.

### Operating system and version

* OS: openSUSE Tumbleweed latest as of November 19,2024
* RSS Guard Version: 4.7.4 (built on Linux/x86_64)
* Revision:
* Build date: 11/20/24 1:35 AM
* KDE Flatpak runtime
* Qt:</b> 6.8.0 (compiled against 6.8.0)

**Comment by richardstevenhack** [comment]
Unfortunately I just set the settings back to use Firefox as a workaround - and now it doesn't work for Firefox anymore.

**Comment by richardstevenhack** [comment]
OK, I fixed it. I don't know why it wasn't working in Firefox, but I managed to fix it by doing the following:

1) flatpak --user override --talk-name=org.freedesktop.Flatpak io.github.martinrotter.rssguard
2) /usr/bin/flatpak-spawn in "Web browser executable" and --host firefox "%1" in Parameters for Firefox (which is exactly what I had it set to before switching to Chrome).
3) /usr/bin/flatpak-spawn in "Web browser executable" and --host google-chrome "%1" in Parameters for Google Chrome.

Now it works in both Firefox and Chrome.

Perhaps the -talk-name got messed up in my flatpak installation, I don't know. I tried all those parameter settings before when it didn't work, but I didn't redo the --talk-name.  So perhaps that fixed it. Or it's just another openSUSE Tumbleweed weirdness.

Anyway we can close this one.
